week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -1,17 +1,15 @@
 // Fix Functions
 
-// Aim: to understand the change code inside functions 
+// Aim: to understand the change code inside functions
 //
 // The below functions are syntactically correct but not outputting the right results.
 // Look at the tests and see how you can fix them.
 
-
-
 // 1) mood function does this return `"I am not happy"`
 //    Only make edits inside the function
 
-function mood() {
-  let isHappy = true;
+function mood(isHappy) {
+  //let isHappy = true;
```

Comment:
This will not work as isHappy is not defined. You also need an 
if ( condition) { 
  return "I am happy";
 } else {
  return "I am not happy";
}